packages/web-components/src/image/image.template.ts

```diff
@@ -0,0 +1,23 @@
+import { ElementViewTemplate, html } from '@microsoft/fast-element';
+import type { Image } from './image.js';
+
+/**
+ * Template for the Image component
+ * @public
+ */
+export const template: ElementViewTemplate<Image> = html<Image>`
+  <template>
+    <img
+      alt="${x => x.alt}"
+      block="${x => x.block}"
+      border="${x => x.border}"
+      borderRadius="${x => x.borderRadius}"
+      fit="${x => x.fit}"
+      margin="${x => x.margin}"
+      role="${x => (x.presentation ? 'presentation' : null)}"
+      shadow="${x => x.shadow}"
+      shape="${x => x.shape}"
+      src="${x => x.src}"
```

Comment:
Were we to take an approach like this in the future, I think you want to bind any of the attributes which aren't native to the img element on the host.